.github/workflows/test_and_deploy.yml

```diff
@@ -53,7 +53,9 @@ jobs:
           PLATFORM: ${{ matrix.platform }}
 
       - name: Coverage
-        uses: codecov/codecov-action@v3
+        uses: coverallsapp/github-action@v2
+        with:
+          path-to-lcov: coverage.xml
```

Comment:
No idea why but this was required https://github.com/coverallsapp/github-action/issues/172#issuecomment-1529837492

Reply:
it looks like `coveralls` found what it thought was two coverage files and tried to read them both (https://github.com/epitools/epitools/actions/runs/4863781567/jobs/8671955880#step:7:64) . It found the actual coverage report called `coverage.xml` plus a file called `.coverage`, which I'm guessing is some kind of configuration file? Perhaps written by `codecov`? Anyway, at least it's working now!

Reply:
Yeah, but when I run it locally I only get `.coverage` so that is the confusing bit. Plus, this was needed for `windows`, not `ubuntu` - not sure about `macOS` as it didn't get to run because `windows` failed.

Reply:
ah okay, no idea then!